src/main/java/org/micks/champmaker/auth/UserPersistApi.java

```diff
@@ -1,6 +1,9 @@
 package org.micks.champmaker.auth;
 
+import java.util.List;
+
 public interface UserPersistApi {
 
     void storeUser(String username, String encryptedPassword);
+    List<User> readFile();
```

Comment:
Rename the method name - this should be more like `getUsers()`

Reply:
fixed